homeassistant/components/doods/manifest.json

```diff
@@ -2,6 +2,6 @@
   "domain": "doods",
   "name": "DOODS - Dedicated Open Object Detection Service",
   "documentation": "https://www.home-assistant.io/integrations/doods",
-  "requirements": ["pydoods==1.0.2", "pillow==7.2.0"],
+  "requirements": ["pydoods==1.0.2", "pillow==8.0.0"],
```

Comment:
The latest release is [8.0.1](https://pillow.readthedocs.io/en/stable/releasenotes/8.0.1.html). This release is fixing CVE-2020-15999.

Reply:
Updated to 8.0.1